### PR TITLE
Area chart component 

### PR DESCRIPTION
**Pull request by blackinno**
- Area chart component 
- Add debounce function in mixins.
- Fix props in mixins file

### CODE REVIEW

File: components/Chart/Pie/PieChart.vue [hunk 1]
```diff
@@ -8,7 +8,7 @@ import chart from '../chart.mixin'
 export default {
   name: 'PieChart',
   mixins: [chart],
-  props: { innerSize: { type: String } },
+  props: { innerSize: { type: String }, animation: { type: Boolean, default: false }, tooltipDescription: { type: String, default: undefined } },
```
Comment (antho1404) [inline components/Chart/Pie/PieChart.vue]: could you put that with new line like in the other components, that will be easier to read

Comment (antho1404) [reply components/Chart/Pie/PieChart.vue]: I don't think `innerSize` is used

Comment (blackinno) [reply components/Chart/Pie/PieChart.vue]: > I don't think `innerSize` is used

`innerSize` is used to make thin or thick for a pie chart. I think it needs to declare `innerSize` for custom a pie chart and set a default value to a pie chart

Comment (blackinno) [reply components/Chart/Pie/PieChart.vue]: > could you put that with new line like in the other components, that will be easier to read

Fixed.

**Pull request merged**